### PR TITLE
Feature/implement show list todo by status

### PR DESCRIPTION
**Pull request by HoangLiarPhuongLe**
- Implement Show list Todos by Status

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -12,6 +12,33 @@
             <h1 class="header-title">todo app</h1>
         </header>
 
+        <section class="search-container">
+            <form class="search-form">
+                <input type="search" name="search" class="input-search" />
+            </form>
+            <button class="btn-search">Search</button>
```
Comment (phuocduong-agilityio) [inline src/index.html]: If you use button for form, it should be button with type submit, and button will locate inside form

File: src/typescript/views/todoView.ts [hunk 2]
```diff
@@ -50,14 +75,45 @@ class TodoView {
...
+        const radioForm: HTMLFormElement = document.querySelector('.radio-button-form')!;
+
+        radioForm.addEventListener('change', async (e) => {
+            e.preventDefault();
+
+            if (e.target instanceof HTMLInputElement) {
+                const selectValue: string = e.target.value;
+
+                switch (selectValue) {
+                    case 'All':
```
Comment (phuocduong-agilityio) [inline src/typescript/views/todoView.ts]: Use enum pls

File: src/index.html [hunk 3]
```diff
@@ -12,6 +12,33 @@
...
+                <input type="radio" name="todos" class="btn-checked" value="Checked" />
+                <label for="todosChecked">Checked</label>
+                <input type="radio" name="todos" class="btn-none" value="None" />
+                <label for="todosNoneChecked">None</label>
+            </form>
+        </div>
+
+        <div class="select-categories-container">
+            <label for="categories">Filter by categories:</label>
+            <select name="categories" class="select-categories-form">
```
Comment (phuocduong-agilityio) [inline src/index.html:37]: For category, it maybe will have more. So should render by JS. It's ok for now. However, next time, for selection, we should use JS for render

File: src/typescript/templates/todoTemplate.ts [hunk 4]
```diff
@@ -20,7 +20,7 @@ class TodoTemplate {
                     <p class="todo-info-text">Category: ${todo.category.name}</p>
                 </div>
                 <form>
-                    <input type="checkbox" />
+                    <input type="checkbox" ${todo.status ? 'checked' : ''} />
```
Comment (phuocduong-agilityio) [inline src/typescript/templates/todoTemplate.ts:23]: If you use form, it should be have form action (Like submit, cancel). For now, it just only a input. So consider use input without form

File: src/index.html (hunk 1 above)
Comment (HoangLiarPhuongLe) [reply src/index.html]: I fixed @phuocduong-agilityio. Please help me check. Thanks!

File: src/typescript/views/todoView.ts (hunk 2 above)
Comment (HoangLiarPhuongLe) [reply src/typescript/views/todoView.ts]: I fixed @phuocduong-agilityio. Please help me check. Thanks!

File: src/typescript/templates/todoTemplate.ts (hunk 4 above)
Comment (HoangLiarPhuongLe) [reply src/typescript/templates/todoTemplate.ts:23]: I fixed @phuocduong-agilityio . Please help me check. Thanks!

**Pull request merged**